install_ubuntu_deps.sh

```diff
@@ -7,6 +7,9 @@ if [ -x "$(command -v sudo)" ]; then
 else
   apt-get install zlib1g-dev
 fi
-curl -L -v -o binaryen.tar.gz https://github.com/WebAssembly/binaryen/releases/download/version_97/binaryen-version_97-x86_64-linux.tar.gz
+
+BINARYEN_VERSION=105
+
+curl -L -v -o binaryen.tar.gz https://github.com/WebAssembly/binaryen/releases/download/version_${BINARYEN_VERSION}/binaryen-version_${BINARYEN_VERSION}-x86_64-linux.tar.gz
 tar xzvf binaryen.tar.gz
-cp binaryen-version_97/bin/* /usr/local/bin
+cp binaryen-version_${BINARYEN_VERSION}/bin/* /usr/local/bin
```

Comment:
Just a trailing newline for consistency
```suggestion
cp binaryen-version_${BINARYEN_VERSION}/bin/* /usr/local/bin

```